### DISCUSSION TITLE
cannot access edit mii interactions

### DISCUSSION THREAD

**Discussion by nintendelia** [Brionjv/Tomodachi-Life-Save-Editor#129] [Q&A]
why can't i access "edit mii interactions?"

**Comment by Brionjv** [comment]
Used version 4.0.4 to edit interactions

**Reply by nintendelia** [reply]
i am, and it still has it blocked.

**Reply by Brionjv** [reply]
it not, if you have a korean or japanese save file, interactions are not listed (you can only edit interactions value in blue).
